.gitignore

```
@@ -127,4 +127,4 @@ venv.bak/
 dmypy.json
 
 # Pyre type checker
-.pyre/
+.pyre/
```

Let's add a newline at the end to keep linters happy
```suggestion
.pyre/

```